source/scripts/_python/README.md

```diff
@@ -0,0 +1,19 @@
+`bundle_lister.py` is a Python (3.6.0) web scraper and file generator. First, it clones the clr-bundles directory, https://github.com/clearlinux/clr-bundles. Second, it parses content all bundles in the clr-bundles/ directory and the `packages-descriptions` file. Third, it uses Jinja2 to output the result of the analysis to: bundles.html.txt. This ``.txt`` file is then referenced in  `bundles.rst`, whose title is `Available bundles`, which is currently: https://clearlinux.org/documentation/clear-linux/reference/bundles.
```

Comment:
Do you want to make the python version 3.6.0+ to include 3.6.0 plus new updates?
Also, should "Second, it parses content all bundles..." be "Second, it parses content of all bundles..."

Reply:
@tpleavitt.  Python 3.6.0 is fine by itself. Modules are shown in `requirements.txt`. Any developer knows to look there.  I'll reword the README as you suggest. Thanks for the careful reading.

Reply:
Sounds good.  With that it looks good to go.  Awesome work on this Michael!  Thank you